packages/vuetify/src/services/theme/index.ts

```diff
@@ -202,6 +214,19 @@ export class Theme extends Service {
     }
   }
 
+  private applyVueMeta23 () {
+    const { set } = this.vueMeta.addApp('vuetify')
```

Comment:
Shouldn't `addApp` need to be called only once ?

Reply:
Both is fine, calling addApp everytime adds only [a little overhead](https://github.com/nuxt/vue-meta/blob/master/src/shared/additional-app.js)

Reply:
Okay, well unless user is making theme update every millisecond it shouldn't affect performance much